vs/README.md

```diff
@@ -0,0 +1,200 @@
+# Build Marian on Windows with GPU support

```

Comment:
Build -> Building? How to build?

Reply:
No, just a grammar fix of this one comment line. Grammatically, it should be either "Building Marian..." or "How to build Marian..."

Reply:
OK. Get it. Thanks